jni/lib/src/errors.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
```

Comment:
2023

Reply:
Hmm, technically it's a file rename for `jexceptions.dart`, do I change the year still?

Reply:
Ah, the GitHub UI is not showing it as a rename to me.

Reply:
`git` doesn't show it as a rename since more than a certain percent of the file has been changed. I can change the year if we're basing our copyright logic on git. https://en.wikipedia.org/wiki/Ship_of_Theseus